include/util/debug_geometry.hpp

```diff
@@ -69,6 +71,8 @@ boost::filesystem::ofstream dg_debug_turns_file;
 bool dg_output_turn_debug = false;
 bool dg_first_turn_debug = true;
 
+std::unordered_map<OSMNodeID, util::FixedPointCoordinate> node_lookup_map;
```

Comment:
Hm actually the same applies to the two bools above. For those we could simply make them `const constexpr auto` in the header and call it a day.